learning/tour-of-beam/learning-content/final-challenge/final-challenge-1/description.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+### Final challenge 1
+
+You have a transaction file with data. You need to parse the csv file. And write a **Pojo class** for it so that Pipeline understands which object it is working with. Make a report that is sent every 30 seconds. Filter so that the **quantity** of the product is more than 20. And divide into 2 parts. In the first part, the transaction **price** should be more than 10, the second less. Combine so that for each transaction id there is a summed price. And write in two files for **"price more than 10"** and **"price less than 10"**.
```

Comment:
Summed price or transaction amount (e.g. price * quantity)? I think totals e.g. sum of price * quantity by ProductNo make more sense

Reply:
Propose following text:
You’re given a csv file with purchase transactions. Write a Beam pipeline to prepare a send report every 30 seconds. The report needs to be created only for transactions where **quantity** is more than 20. 

Report should consist of two files named **"price more than 10"** and **"price less than 10"**:
- Total transactions amount grouped by **ProductNo** for products with **price** greater than 10
- Total transactions amount grouped by **ProductNo** for products with **price** greater than 10